### PR TITLE
Initalize

### PR DESCRIPTION
**Pull request by shah-shuja-dev**
## This Branch Implements The Following : 

- Project Setup and Initialization.
- Installing all the packages.
- Installing All The Linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,47 @@
...
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+To get a local copy up and running follow these simple example steps.
```
Comment (danmainah) [inline README.md]: - [ ] Kindly add a way to clone your repo, it wi make it easier for amateur users know how to copy your project and run it locally.

File: README.md [hunk 2]
```diff
@@ -64,12 +55,9 @@ Give a ⭐️ if you like this project!
...
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- The Awesome Teamwork! and Microverse!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```
Comment (danmainah) [inline README.md:63]: - [ ] [optional] Kindly personalize your MIT license, it will make it be professional.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shuja-shah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by gemmen29 — APPROVED** [review]
Hi @shuja-shah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**